src/js/tabs/send.controller.js

```diff
@@ -1132,6 +1148,11 @@ SendTab.prototype.angular = function (module)
       // Add memo to tx
       tx.addMemo('client', 'text/plain', 'rt' + $scope.version);
 
+      // Travel rule data
+      if (send.travelRuleData) {
+        tx.addMemo('travelrule', 'text/plain', $scope.travelRuleData);
```

Comment:
Not sure what ripple-lib the client uses, but it looks like lib is moving towards using `options` objects in methods: https://github.com/ripple/ripple-lib/blob/develop/src/core/transaction.js#L821.  Just something to be aware of.